### ISSUE TITLE
Getting ERR_SSL_VERSION_OR_CIPHER_MISMATCH

### ISSUE THREAD

**Issue by xarnze** [OptimalBits/redbird#91]
I've just tried to setup LetsEncrypt with Redbird but all I get in the browser is ERR_SSL_VERSION_OR_CIPHER_MISMATCH 

Any Ideas?

**Comment by manast** [comment]
not really, but you could try different versions of node, I have succeeded with 4.x in production.

**Comment by b3nz0** [comment]
@xarnze I was getting this error early on and it was due to LetsEncrypt not generating/fetching the certificate correctly.

Think my issue was resolved with verifying permissions to the certificate location.

My config starts with;

var proxy = require('redbird')({
  port: 80,
  xfwd: false,
  letsencrypt: {
    path: "/home/user/redbird/certs",
    port: 3000
  },
  ssl: {
    port: 443,
    http2: false,
  }
});

Then for each service I want to proxy;

proxy.register("host.example.com", "https://1.2.3.4:1234", {
    ssl: {
    letsencrypt: {
      email: "me@domain.com",
      production: true
      }
    }
});

**Comment by mjugger** [comment]
I am also getting this error.
@xarnze 
what did you mean by "verifying permissions to the certificate location"?

**Comment by animatedcreativity** [comment]
Yes, a permission issue:
On MacOSX certbot had to run as root only using sudo and certificates generated were saved as root user, so redbird node instance was unable to access them.
Used rsync to copy certificates to another folder and changed permissions using chown.
